docs/docs-content/integrations/verified_packs.md

```diff
@@ -31,3 +31,51 @@ To learn about our general pack maintenance policy, review the [Maintenance Poli
 Palette paid subscriptions cover access to our Support team and product updates.
 
 :::
+
+## Prerequisites
+
+- Your Palette account role must have the `clusterProfile.create` permission to create a profile. Refer to the
+  [Roles and Permissions](../user-management/palette-rbac/project-scope-roles-permissions.md#cluster-profile-admin)
+  documentation for more information.
+
+## Use Verified Packs
+
+Palette comes with a default pack registry that is available to all SaaS tenants, and non-airgap self-hosted Palette
+environments. The [Default Registries](../registries-and-packs/registries/registries.md#default-registries) table has
+the details of the Palette registry. You can use verified packs in your cluster profiles without any special
+configuration.
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+2. From the left **Main Menu** click **Profiles**.
+
+3. Click on the **Add Cluster Profile** button.
+
+4. Fill out the cluster profile name and description.
+
+5. Select **Add-on** for the cluster profile type. Click on **Next** to continue.
+
+6. Select **Add New Pack**. In the next window that displays, choose the **Palette Registry** and select the pack to add
```

Comment:
We don't have a filter in the product to select verified packs so as mentioned above, they need to pick a pack with the verified icon badge
![CleanShot 2024-05-23 at 13 03 20](https://github.com/spectrocloud/librarium/assets/29551334/d737d77d-8763-41f3-927c-a6503688927f)

Reply:
Called out the verified badges in this step and changed the screenshot to highlight the badges too.